jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -808,6 +808,7 @@ def apply_RGB_presets(self):
             cmap = matplotlib.colormaps['gist_rainbow'].resampled(n_visible)
             preset_colors = [matplotlib.colors.to_hex(cmap(i), keep_alpha=True) for
                              i in range(n_visible)]
+            preset_colors.reverse()

```

Comment:
Why not reverse it in the list comprehension? Also why is reverse needed here to fix the bug?
```suggestion
                             i in range(n_visible + 1, -1, -1)]
```

Reply:
or just `cmap(n_visible - i)` 🤷‍♂️  I don't think any of these are expensive, so whatever you think is most readable (and an inline comment explaining the reason for reversal might help future-us)

Reply:
I pushed a change but it seems like this PR didn't pick it up? I wonder if Github PRs are on the fritz right now.